### PR TITLE
Css grid layout

### PR DESCRIPTION
**Pull request by mikolaslorant**
## Summary

-added navbar shadow
-css tech component color change to white
-added css grid layout for tech component
## Screenshot

css-grid

## Trello cards
-Rediseño de la página

### CODE REVIEW

File: scss/navbar.scss [hunk 1]
```diff
@@ -1,10 +1,12 @@
 .navbar {
   background-color: white;
-  border-bottom: 2px solid $grey;
   height: 70px;
   position:fixed;
   width: 100%;
   z-index: 1;
+  -webkit-box-shadow: 0px 6px 27px -7px rgba(0,0,0,0.55);
```
Comment (FrankIglesias) [inline scss/navbar.scss]: use variable color for the last param

File: scss/tech.scss [hunk 2]
```diff
@@ -50,11 +29,19 @@
...
+  grid-template-columns: 220px 220px 220px;
+  grid-template-rows: 320px 320px;
+  justify-content: center;
   margin: auto;
-  flex-direction: row;
-  flex-wrap: wrap;
-  justify-content: flex-start;
   padding-left: 50px;
+  width: 100%;
+
```
Comment (FrankIglesias) [inline scss/tech.scss]: remove this line

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by FrankIglesias** [comment]
Excellent job

**Comment by mikolaslorant** [comment]
Corrected requested changes.

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**